src/scenarios/MultilateralFederation.md

```diff
@@ -0,0 +1,60 @@
+## Multilateral Federation - WAYF Discovery Services
+Supporting the use of a central SAML IdP Discovery and Persistence service in a multilateral federation scenario.
+
+### Summary
+
+#### Contributor 
+- Name: Heather Flanagan
+- Organization: Spherical Cow Consulting
+- Email: hlf@sphericalcowconsulting.com
+- Name: Nicole Roy
+- Organization: University Corporation for Advanced Internet Development dba Internet2
+- Email: nroy@internet2.edu
+- Name: Leif Johansson
+- Organization: SUNET
+- leifj@sunet.se
+
+#### Protocol
+- Name: SAML
+- Grant/flow (if applicable): This scenario makes use of the SAML front-channel bindings defined in OASIS SAML v2.0, specifically the Web Browser SSO Profile, Identity Provider Discovery Profile, and on rare occasions, Single Logout Profile contained in the "Profiles for the OASIS SecurityAssertion Markup Language (SAML)V2.0". Additional context for the use of these profiles and flows in the multilateral SAML federation space used in the global research and education sector is specified in the Kantara Initiative profiles. 
+- Reference: <https://www.oasis-open.org/committees/download.php/35389/sstc-saml-profiles-errata-2.0-wd-06-diff.pdf>, <https://kantarainitiative.github.io/SAMLprofiles/fedinterop.html>, <https://kantarainitiative.github.io/SAMLprofiles/saml2int.html>
+
+#### Browser Features Required
+- 1st party Cookie
+- 3rd party cookies
+- Redirect with link decoration
+- Persistent Local Storage
+- cross-domain postMessage
```

Comment:
Would it be possible to highlight in the 'Description of flow' section how each of these are used? The use of local storage is already well described. I think it would be helpful for the scenario to highlight how 3rd party cookies are required as well as cross-domain postMessage.

Reply:
I think I went overboard with the pieces that are required for this specific use case. I've updated the file and updated the drawings to try and be more clear about exactly what/where the different components interact with browser features.